src/drivers/imx/sdma.c

```diff
@@ -478,11 +478,22 @@ static int sdma_start(struct dma_chan_data *channel)
 {
 	struct sdma_chan *pdata = dma_chan_get_data(channel);
 
-	if (channel->status == COMP_STATE_INIT)
-		return -EINVAL;
-
 	trace_sdma_error("sdma_start(%d)", channel->index);
 
+
```

Comment:
@dbaluta nothing to do with this patch. But the trace above is not really an error is it?

Reply:
@ranj063 indeed it is not an error. It belongs to a separate patch.